packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -34,11 +34,13 @@ const Backdrop = React.forwardRef(function Backdrop(props, ref) {
     invisible = false,
     open,
     transitionDuration,
+    // eslint-disable-next-line react/prop-types
+    TransitionComponent = Fade,
     ...other
   } = props;
 
   return (
-    <Fade in={open} timeout={transitionDuration} {...other}>
+    <TransitionComponent in={open} timeout={transitionDuration} {...other}>
```

Comment:
Oops, I had forgotten about this one in https://github.com/mui-org/material-ui/issues/17047#issuecomment-523549825.